src/applications/auth/manifest.json

```diff
@@ -4,5 +4,10 @@
   "entryName": "auth",
   "rootUrl": "/auth/login/callback",
   "noRouting": true,
-  "production": true
+  "production": true,
```

Comment:
Our build doesn't do anything with the `production` property, but I'm leaving it anyway in case the app itself uses that for something (ie determining whether to run e2e tests)